src/Testing/src/xunit/SkipOnHelixAttribute.cs

```diff
@@ -42,6 +42,19 @@ public string SkipReason
                 return $"This test is skipped on helix";
             }
         }
+        
+        private bool ShouldSkip()
+        {
+            if (Queues == null) 
+            {
+                return true;
+            }
+            if (Queues.Contains("All.OSX")) 
```

Comment:
nit: Suggest adding a variable set above this line containing `GetTargetHelixQueue().ToLowerInvariant()`. Also slightly prefer a blank line after the previous block.